cypress/support/components/ActionModal.js

```diff
@@ -26,6 +29,48 @@ export function TypeResponse(textToType) {
   })
 }
 
+export function TypeApiLogicArgs(args) { TypeApiArgs('Logic Arguments', args) }
+export function TypeApiRenderArgs(args) { TypeApiArgs('Render Arguments', args) }
+
+function TypeApiArgs(apiArgLabel, args) {
+  cy.Get('label')
+    .contains(apiArgLabel)
+    .siblings('div.editor-container')
+    .find('div[data-slate-editor="true"]')
+    .then(elements => {
+      if (elements.length != args.length) {
+        throw new Error(`Test Code Error: The API ${apiArgLabel} takes ${elements.length} arguments, but test code supplied ${args.length}`)
+      }
+
+      cy.DumpHtmlOnDomChange(true)
```

Comment:
While this may be possible, I expect that it will be complex and take more time than I want it to. This code executes in the background inside of MonitorDocumentChanges.js, so I don't want it to slow the tests down. I turn it on when I need it and turn it off when I don't. It gets logged to the log file and I use external tools, BeyondCompare being the primary one, to compare the differences when I need to.